modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/shipment/test/CommerceShipmentItemTest.java

```diff
@@ -362,7 +362,7 @@ public void testUpdateCommerceShipmentItemOnShippedShipment()
 
 		CommerceShipmentItem newCommerceShipmentItem =
 			_commerceShipmentItemLocalService.updateCommerceShipmentItem(
-				_commerceShipmentItem.getCommerceShipmentItemId(),
+				_commerceShipmentItem.getCommerceShipmentItemId(), null,
```

Comment:
why here we cannot use _commerceShipmentItem.getExternalReferenceCode() ? 
In case is not possible in the *Test classes please use a random string

Reply:
That would be the warehouses' ERC, so I'll use randomString()